src/EnergyPlus/HVACDXHeatPumpSystem.hh

```diff
@@ -215,27 +206,18 @@ struct HVACDXHeatPumpSystemData : BaseGlobalStruct
         this->DXHeatPumpSystem.deallocate();
         this->QZnReq = 0.001;
         this->QLatReq = 0.0;
-        this->MaxONOFFCyclesperHour = 4.0;
-        this->HPTimeConstant = 0.0;
-        this->FanDelayTime = 0.0;
         this->OnOffAirFlowRatio = 1.0;
         this->ErrorsFound = false;
         this->TotalArgs = 0;
         this->MySetPointCheckFlag = true;
         this->SpeedNum = 1;
         this->QZnReq = 0.001;
         this->QLatReq = 0.0;
-        this->MaxONOFFCyclesperHour = 4.0;
-        this->HPTimeConstant = 0.0;
-        this->FanDelayTime = 0.0;
         this->OnOffAirFlowRatio = 1.0;
         this->SpeedRatio = 0.0;
         this->SpeedNumber = 1;
         this->QZoneReq = 0.001;
         this->QLatentReq = 0.0;
-        this->MaxONOFFCyclesperHr = 4.0;
-        this->HPTimeConst = 0.0;
-        this->HPFanDelayTime = 0.0;
```

Comment:
A lot of weird things in this old code that can now definitely be removed with this change.